.vscode/tasks.json

```diff
@@ -0,0 +1,61 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Build (Jekyll)",
+      "type": "shell",
+      "command": "bundle",
+      "args": [
+        "exec",
+        "jekyll",
+        "build"
+      ]
```

Comment:
The build command is missing the required environment variable `JEKYLL_ENV=production`, see: <https://chirpy.cotes.page/posts/getting-started/#manual-build-and-deployment>